lhotse/bin/modes/recipes/__init__.py

```diff
@@ -56,3 +56,4 @@
 from .voxceleb import *
 from .wenet_speech import *
 from .yesno import *
+from .vcc2018 import *
```

Comment:
We try to keep the imports sorted in lexicographical order. Could you put this import above `vctk`?